client/data/reader/use-related-sites.ts

```diff
@@ -50,15 +50,15 @@ const selectRelatedSites = ( response: { sites: Site[] } ): RelatedSite[] | null
 
 export const useRelatedSites = ( siteId: number ): UseQueryResult< RelatedSite[] | null > => {
 	const site_recs = 5;
-	const path =
-		'/read/site/' + siteId + '/sites/related?size_global=' + site_recs + '&http_envelope=1';
```

Comment:
Are we intentionally leaving out the `http_envelope=1` arg?

Reply:
Yes. My guess is this was copied from the actual request URL. This part is usually added by the wpcom request library.

Reply:
Hmm, interesting. I find a difference between the requests done on this branch and production though:

Production:
![Screenshot 2023-06-01 at 13 41 36](https://github.com/Automattic/wp-calypso/assets/8436925/6d962409-b38d-4730-a835-2c0e67a84505)

This branch:

![Screenshot 2023-06-01 at 13 41 30](https://github.com/Automattic/wp-calypso/assets/8436925/b4c5c061-bbd8-42f3-9138-d902c347dc80)

I expected that the requests will be the same. For the protocol, I'm testing on a reader full post page.

Reply:
Seems like it needs to be added as we're not going through the legacy data layer (which I think adds it for you if needed). I did some minor tweaks and rebased onto current trunk to fix the conflicts.

Reply:
That does the job, thanks 👍